scripts/start_civ.sh

```diff
@@ -475,6 +479,7 @@ function set_pt_usb() {
             echo "passthrough USB device: $d"
             set_pt_pci_vfio $d
             GUEST_USB_PT_DEV+=" -device vfio-pci,host=${d#*:},x-no-kvm-intx=on"
+	    PT_ENABLE="true"
```

Comment:
remove it. 
add logic below for-loop: if GUEST_USB_PT_DEV != "", then GUEST_USB_XHCI_OPT=""

Reply:
ACK